proj/src/main/webapp/index.html

```diff
@@ -28,14 +28,12 @@
             <button id="btnLogout" class="hidden">Log out</button>
         </div>
 
-        <div id="content"></div>
-        <ul id="chat-as-list"></ul>
-        <input type="text" id="message-input" />
-        <button type="button" onclick="pushChatMessage()" id="send-button">send</button>
-        <div id="content"></div>
-        <div id="messages"></div>
         <a href="/static/welcome.html">Welcome Page</a>
+        <a href="/static/chat.html">Chat Page</a>
 
+        <!-- <ul id="chat-as-list"></ul>
+        <input type="text" id="message-input" />
+        <button type="button" onclick="pushChatMessage()" id="send-button">send</button> -->
```

Comment:
Can you remove this commented out bit if it isn't being used.

Reply:
Just removed it